core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitorSpec.scala

```diff
@@ -94,26 +93,42 @@ class V1WorkspaceMigrationMonitorSpec
         )
       }
 
-      val (_, _, dbOp) = Await.result(
+      val (_, _, dbOp) = IO.fromFuture(IO {
         services.slickDataSource.database
           .run {
             V1WorkspaceMigrationMonitor.migrations
               .filter(_.workspaceId === spec.testData.v1Workspace.workspaceIdAsUUID)
               .result
           }
-          .map(_.head)
-          .flatMap { attempt =>
-            V1WorkspaceMigrationMonitor.claimAndConfigureNewGoogleProject(
-              attempt,
-              services.workspaceService,
-              spec.testData.v1Workspace,
-              spec.testData.billingProject
-            )
-          },
-        5.seconds // really should be instant if we've mocked correctly?
-      )
+      })
+        .map(_.head)
+        .flatMap { attempt =>
+          V1WorkspaceMigrationMonitor.claimAndConfigureNewGoogleProject(
+            attempt,
+            services.workspaceService,
+            spec.testData.v1Workspace,
+            spec.testData.billingProject
+          )
+        }
+        .unsafeRunSync
 
       spec.runAndWait(dbOp) shouldBe ()
+
+      val (projectId, projectNumber, projectConfigured) = IO.fromFuture(IO {
+      services.slickDataSource.database
+        .run {
+          V1WorkspaceMigrationMonitor.migrations
+            .filter(_.workspaceId === spec.testData.v1Workspace.workspaceIdAsUUID)
+            .map(r => (r.newGoogleProjectId, r.newGoogleProjectNumber, r.newGoogleProjectConfigured))
+            .result
+        }
+    })
+      .map(_.head)
+      .unsafeRunSync
+
+      projectId should be ('defined)
+      projectNumber should be ('defined)
+      projectConfigured should be ('defined)
```

Comment:
I think this would work too:

```suggestion
      projectConfigured shouldBe defined
```

Reply:
Nice! Thanks